### PR TITLE
Prep Javascript Objects First Draft Nelson Daza

### PR DESCRIPTION
**Pull request by neldaza**
Screen Shot 2021-09-22 at 11 21 05 AM

### CODE REVIEW

File: prep-javascript-objects/main.js [hunk 1]
```diff
@@ -0,0 +1,19 @@
+var person = {
+  firstName: 'John',
+  lastName: 'Smith',
+  hobby: 'baseball',
+  job: 'plumber',
+  previousJob: 'teacher'
```
Comment (scbowler) [inline prep-javascript-objects/main.js]: We don't want to set these here, they should be set later in the code using dot notation. The idea is to learn different ways of setting a property on an object.

File: prep-javascript-objects/main.js [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+console.log(person);
+var fullName = 'John ' + 'Smith';
+var nameDeclare = "The person's full name is";
+console.log(nameDeclare, fullName);
+var currentJob = 'plumber';
+var currentJobDeclare = "The person's current job is";
+console.log(currentJobDeclare, currentJob);
+var previousJob = 'teacher';
+var previousJobDeclare = "The person's previous job was";
+console.log(previousJobDeclare, previousJob);
```
Comment (scbowler) [inline prep-javascript-objects/main.js:17]: Down in this area is where you would use dot notation to set the `job` and `previousJob` properties on the `person` object.

**Review by scbowler — CHANGES_REQUESTED** [review]
Looking good, just a couple small updates needed. Let me know if you have any questions.

File: prep-javascript-objects/main.js (hunk 2 above)
Comment (neldaza) [reply prep-javascript-objects/main.js:17]: done!
Screen Shot 2021-09-22 at 4 07 47 PM

**Review by scbowler — APPROVED** [review]
Looks good! 👍

**Pull request merged**